Content.Shared/Hands/EntitySystems/SharedHandsSystem.Drop.cs

```diff
@@ -109,8 +109,12 @@ public bool TryDrop(EntityUid uid, Hand hand, EntityCoordinates? targetDropLocat
         if (!CanDropHeld(uid, hand, checkActionBlocker))
             return false;
 
+        var userXform = Transform(uid);
+        var isInContainer = ContainerSystem.IsEntityOrParentInContainer(uid, xform: userXform);
+
         var entity = hand.HeldEntity!.Value;
-        DoDrop(uid, hand, doDropInteraction: doDropInteraction, handsComp);
+        if (!isInContainer)
+            DoDrop(uid, hand, doDropInteraction: doDropInteraction, handsComp);
```

Comment:
Isn't this just PlaceNextTo?

Reply:
Apparently this was caused by a bulk replace, so I need to actually dig through and figure out whats going on better, I just figured DropNextTo was there for a good reason. I'll update.